Makefile

```diff
@@ -75,58 +55,6 @@ metadata-docker-publish: build ## publish metadata mock-server linux/amd64 platf
 metadata-docker-run: metadata-docker-build ## build and run metadata mock-server linux/amd64 platform docker image (watching /data/*.json)
 	@docker run --rm -p 3000:3000 bazel/services/metadata/backend/fake-backend:image
 
-.PHONY: metadata-service-run
-metadata-service-run: build ## start the metadata-service
-	@bazel run //services/metadata/backend/cmd
-
-.PHONY: metadata-service-test
-metadata-service-test: ## run all metadata-service tests
-	@bazel test //services/metadata/backend/...
```

Comment:
this one is for running the metadata service tests and will be one of the most run targets when you start working on the BE part.